### PR TITLE
Add perf improvements to 3.0 release notes

### PR DESCRIPTION
**Pull request by shirhatti**

### CODE REVIEW

**Comment by davidfowl** [comment]
@shirhatti you da real MVP

File: aspnetcore/release-notes/aspnetcore-3.0.md [hunk 1]
```diff
@@ -482,6 +482,14 @@ In previous versions of ASP.NET Core, calling <xref:Microsoft.AspNetCore.Builder
 
 This scenario is fixed in ASP.NET Core 3.0. The host enables the [Forwarded Headers Middleware](xref:host-and-deploy/proxy-load-balancer#forwarded-headers-middleware-options) when the `ASPNETCORE_FORWARDEDHEADERS_ENABLED` environment variable is set to `true`. `ASPNETCORE_FORWARDEDHEADERS_ENABLED` is set to `true` in our container images.
 
+## Performance improvements across the stack
+
+As part of ASP.NET Core 3.0, we've made improvements in reducing memory allocations and improving throughput through various parts of the stack including, but not limited to:
+- Reduction in memory usage when using the built-in Dependency Injection container for scoped services
+- Reduction in allocations across the stack including middleware scenarios and routing.
+- Reduction in memory usage for WebSocket connections.
+- Memory reduction and throughput improvements for HTTPS connections.
```
Comment (davidfowl) [inline aspnetcore/release-notes/aspnetcore-3.0.md]: - New optimized and fully asynchronous JSON serializer.
- Huge memory and throughout improved form parsing.

**Comment by davidfowl** [comment]
Random: Do we have any mention of the new distributed tracing support?

**Comment by shirhatti** [comment]
> Random: Do we have any mention of the new distributed tracing support?

Sadly, no😭

**Comment by davidfowl** [comment]
Add it! First class support for the distributed tracing using the new w3c trace context standard

**Comment by guardrex** [comment]
@shirhatti Except for "call stack" and "stack trace," we don't use "stack" very much. Ok to use "framework" instead? If not, I'll convert it back to "stack."

**Comment by shirhatti** [comment]
s/throughout/throughput

File: aspnetcore/release-notes/aspnetcore-3.0.md [hunk 2]
```diff
@@ -482,6 +482,17 @@ In previous versions of ASP.NET Core, calling <xref:Microsoft.AspNetCore.Builder
 
 This scenario is fixed in ASP.NET Core 3.0. The host enables the [Forwarded Headers Middleware](xref:host-and-deploy/proxy-load-balancer#forwarded-headers-middleware-options) when the `ASPNETCORE_FORWARDEDHEADERS_ENABLED` environment variable is set to `true`. `ASPNETCORE_FORWARDEDHEADERS_ENABLED` is set to `true` in our container images.
 
+## Performance improvements
+
+Throughout the framework, ASP.NET Core 3.0 includes many improvements that reduce memory allocations and improve throughput:
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-3.0.md]: ASP.NET Core 3.0 includes many improvements that reduce memory usage and improve throughput:

Comment (guardrex) [reply aspnetcore/release-notes/aspnetcore-3.0.md]: I left the opener for emphasis on the work.

**Pull request merged**